package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cache",
-  "version": "1.1.2",
+  "version": "2.1.0",
```

Comment:
Why are we bumping the cache action version?

Reply:
Sorry if I'm not understanding, but shouldn't this match the tag / release version?  I'm planning to release this change as `@v2.1`